src/i18n.ts

```diff
@@ -0,0 +1,21 @@
+import i18n from "i18next";
+import { initReactI18next } from "react-i18next";
+import LanguageDetector from "i18next-browser-languagedetector";
+import Backend from "i18next-http-backend";
+
+export enum Languages {
+    Ru = "ru",
+    En = "en",
+}
```

Comment:
I propose to move language constants to the global constants folder so they become accessible from different modules/components (we also need them in the Header for the dropdown language selector)

Reply:
i add plugin to detect browser language. I think it should resolve this problem